Library/Formula/syntaxerl.rb

```diff
@@ -0,0 +1,17 @@
+class Syntaxerl < Formula
+  homepage "https://github.com/ten0s/syntaxerl"
+  url "https://github.com/ten0s/syntaxerl/archive/0.7.0.tar.gz"
+  sha256 "0011fa629e268200f1767b260be984ac6ce83200ac9b401df3d6565cf75ccb24"
+
+  depends_on "erlang"
+
+  def install
+    system "make"
+    bin.install "syntaxerl"
+  end
+
+  test do
+    # syntax-check its own source code
+    system "syntaxerl", "src/syntaxerl.erl"
```

Comment:
If ` src/syntaxerl.erl ` exists in the `buildpath` the test won't have access to it FYI.